tests/test_checkpoint_io/test_low_level_zero_checkpoint_io.py

```diff
@@ -44,10 +45,16 @@ def check_low_level_zero_checkpointIO(stage: int, shard: bool, offload: bool):
     with shared_tempdir() as tempdir:
         model_ckpt_path = f"{tempdir}/model"
         optimizer_ckpt_path = f"{tempdir}/optimizer"
+        if use_async:
+            model_ckpt_path = f"{model_ckpt_path}.safetensors"
+        if not use_async:
+            model_ckpt_path = f"{model_ckpt_path}.pt"
```

Comment:
If shard, it should be `xxx/model`. If not shard and not async, it should be `xxxx/model.pt`. If not shard and async, it should be `xxx/model.safetensors`